.github/actions/next-cloud-release-version/get_next_release_version.py

```diff
@@ -19,8 +19,15 @@ def increment_latest_version(released_versions: List[str], target_version: str)
     target_version = Version.coerce(target_version)
     latest_version = target_version
     latest_version.prerelease = ("post0",)
+
+    # Function to extract the numeric part from prerelease tags like 'post01' or 'post2'
```

Comment:
minor nits:

- This is technically a post release, not a prerelease, which may be confusing to newcomers
- Could we move the `.prerelease[0]` piece here?
- Could we rename this to something shorter, like `def post(version: Version) -> int:`?